client/app/constants/app_constants.coffee

```diff
@@ -92,8 +96,7 @@ module.exports =
         'SETTINGS_UPDATE_FAILURE'   : 'SETTINGS_UPDATE_FAILURE'
 
         # Tasks
-        'RECEIVE_TASK_UPDATE'       : 'RECEIVE_TASK_UPDATE'
-        'RECEIVE_TASK_DELETE'       : 'RECEIVE_TASK_DELETE'
+        'CLICKED_TASK_OK'           : 'CLICKED_TASK_OK'
```

Comment:
What does that mean? I'm not sure that using `clicked` is a good idea: it's about how the event was called not what should be done next. Shouldnt all event name follow this 2nd pattern?